WcaOnRails/app/views/search_results/index.html.erb

```diff
@@ -32,7 +32,11 @@
     <% end %>
 
     <% if !@persons.empty? %>
-      <h2><%= anchorable "People" %></h2>
+      <h2>
+        <%= anchorable "People" %>
+        |
```

Comment:
Nit: I love the look of this, but it looks kind of yucky on hover:

![image](https://cloud.githubusercontent.com/assets/277474/15635922/75f7ff1e-25a3-11e6-93a9-98de9952de33.png)

Maybe just darken or let it become orange on hover?